assets/js/components/notifications/UnsatisfiedScopesAlert.js

```diff
@@ -131,7 +151,10 @@ export default function UnsatisfiedScopesAlert() {
 		'Site Kit can’t access necessary data',
 		'google-site-kit'
 	);
-	const ctaLabel = __( 'Redo setup', 'google-site-kit' );
+	let ctaLabel = __( 'Redo setup', 'google-site-kit' );
+	if ( tempPersistedPermissionsError?.data ) {
+		ctaLabel = __( 'Grant permission', 'google-site-kit' );
+	}
```

Comment:
Personally I'd prefer to avoid this unnecessary `let` with a ternary:

```suggestion
	const ctaLabel = tempPersistedPermissionsError?.data
		? __( 'Grant permission', 'google-site-kit' )
		: __( 'Redo setup', 'google-site-kit' );
```